pkg/controllers/infracluster/infracluster_controller.go

```diff
@@ -19,18 +19,14 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"net/url"
-	"strconv"
 
 	"github.com/go-logr/logr"
 
 	cerrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
This is traditionally imported as `kerrors`, may want to fix at some point separately

Reply:
Updated